src/os/bluestore/BlueStore.cc

```diff
@@ -8587,7 +8600,10 @@ void BlueStore::_deferred_aio_finish(OpSequencer *osr)
       auto q = deferred_queue.iterator_to(*osr);
       deferred_queue.erase(q);
     } else if (deferred_aggressive) {
-      _deferred_submit(osr);
+      dout(20) << __func__ << " queuing async deferred_try_submit" << dendl;
+      finishers[0]->queue(new FunctionContext([&](int) {
+	    deferred_try_submit();
+	  }));
```

Comment:
if we use async 'deferred_try_submit' here, so we actually don't need introduce 'deferred_submit_lock'? Because we'll not block in the aio completion handle.